pages/background.md

```diff
@@ -1,25 +1,49 @@
+
+* _Demand-Pull_ - This is the job availability in the United States when the wages are a significant difference than Mexico's. 
+* _Supply-Push_ - This is the cause of economic stability in Mexico's economy or others that drive people out of the country.
+* _Networks_ - Friends and Family that spark migration from existing residents of the United States
+
+With these categories this can be the base for looking at trends in data. Aguila also shows arguments proving that even low employment rates in Mexico along with decreasing poverty rates still showed a relatively steady pace of migration. Finally, her arguments with networks also serve as a key to migration as her data shows the growth of established communities creating social ties to employments and aiding new migrations to the same occupation. These occupations grow in sync with the growing wage workers and assist in the United States GDP over time. The data and explanations by Aguila are patterns and trends that can be applied to other population growths such as in European populations in the north east and Asian populations in the west of the United States of America. 
+
+---
 
-You have freedom to roam from the narrow topic of your project to explore how different authors/schools of thought have impacted scholarly approaches over time. However, make sure that in the end you clearly relate how this discussion relates to the subject of your project and/or your choices in methods or interpreative models.
+A large basis for this project was to look at how immigration laws affected the migrant flow over time for the United States. Liskofsky points out in their journal of the roots of immigration that is plagued with racism and "Nordic" superiority over other countries. Historically, The United States has shown resentment towards every newcomer as Liskofsky argues. From Catholic's and Jew's to eventually the Chinese where we first witness exclusionary acts in 1882. This leads into further restrictions seen by the Japanese as well before the second world war which is why our data shows such staggering declines in immigration during the period. Following this period, Liskofsky disussed multiple displaces and refugee acts that would bring immigration back to the states. 
```

Comment:
Great, way to be clear about the historiographical basis for this project! Again, add citations, even if you mention the authors by name in the main text. If it feels weird to just have a number like this (3), instead you could do something like this ` ([Bibliography](bibliography.md), 3) `

I will stop mentioning citations from here forward.

Additional note: if you end up examining some of these claims (like claims about Jews or Japanese) in your visual analysis, go ahead and hint at that here (I haven't look at the analysis part as of reading this yet).